streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -1139,8 +1140,7 @@ private <K, V> String readWindowedKeyedMessagesViaConsoleConsumer(final Deserial
                 "--property", "key.deserializer.window.size.ms=500",
             };
 
-            ConsoleConsumer.messageCount_$eq(0); //reset the message count
```

Comment:
Why this is not needed anymore with the new code?

Reply:
I made `ConsoleConsumer.run()` reset to count to 0 in https://github.com/apache/kafka/pull/15274/files#diff-84e7d94a76d5ecef71ab8b1b978972bcfbe869ace82a87086bdf0067ebf6c2fbR70

Reply:
Ah right!